frontend/src/diagram/operations.ts

```diff
@@ -295,6 +295,22 @@ export const invokeEdgeToolOnDiagramMutation = gql`
   }
 `;
 
+export const invokeDeleteToolOnDiagramMutation = gql`
+  mutation invokeDeleteToolOnDiagram($input: InvokeDeleteToolOnDiagramInput!) {
+    invokeDeleteToolOnDiagram(input: $input) {
+      __typename
+      ... on InvokeDeleteToolOnDiagramSuccessPayload {
+        diagram {
+          id
+        }
```

Comment:
I am pretty sure that nobody is using that and that it only exists because others are doing the same mistake again and again